addons/mail/static/tests/tours/mail_activity_schedule_from_chatter.js

```diff
@@ -15,12 +15,14 @@ registry.category("web_tour.tours").add("mail_activity_schedule_from_chatter", {
         {
             trigger: ".dropdown-item:contains('Call')",
         },
+        {
+            trigger: "input[id*='activity_type_id']:propValue('Call')",
+            run: () => {},
+        },
         {
             extra_trigger: "button:contains('Schedule')",
             trigger: "input[id*='activity_type_id']",
-            run: function (action_helper) {
-                setTimeout(() => action_helper.click(), 100);
-            },
+            run: "click",
```

Comment:
click is the default action, it doesn't need to be specified

Reply:
Not for input